mandatory/3-function-output.js

```diff
@@ -3,12 +3,17 @@ function getRandomNumber() {
   return Math.random() * 10;
 }
 
+// Math.random() can be used to return random integers.
```

Comment:
Great, but some more details would be useful. Such as...

What are the parameters of the random integer? 0-9, or 0-infinity?

What specifically is your function doing here? What will it return?